.github/workflows/ci.yml

```diff
@@ -218,6 +218,11 @@ jobs:
       - name: License Scan
         run: |
           /root/executables/ci-utility check --sbom bom.xml --allowedLicences ${{ inputs.allowedLicenses }}
+        env:
+            ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
kindly test the license scan changes without ACCESS_TOKEN variable. And can these be set as global variables?